isilon_create_users.sh

```diff
@@ -289,15 +340,19 @@ case "$DIST" in
         [ $? -ne 0 ] && addError "Could not add user impala$CLUSTER_NAME to hive$CLUSTER_NAME group in zone $ZONE"
 
         #Set some varliables to take these special case group assignments to the group file
-        sqp=`grep sqoop$CLUSTER_NAME $grpfile`
-        hve=`grep hive$CLUSTER_NAME $grpfile`
+        sqp=`grep "\bsqoop$CLUSTER_NAME\b": $grpfile`
```

Comment:
Nice... This looks like it should work :+1:

We weren't adding `sqoop2` to the `sqoop` group before, though, should we be doing that in other paths?